packages/zoe/src/contractSupport/interest.js

```diff
@@ -0,0 +1,60 @@
+// @ts-check
+
+import {
+  floorDivideBy,
+  floorMultiplyBy,
+  invertRatio,
+  multiplyRatios,
+  ratiosSame,
+} from './ratio.js';
+
+/**
+ *
+ * @param {Ratio} currentCompoundedInterest as coefficient
```

Comment:
@Chris-Hibbert WDYT of having a type specifically for "interest rate" so that this qualification isn't necessary? It could also make it impossible for the numerator and denominator to have different brands.

Reply:
[after much out-of-band discussion]

I'm still not clear on what "as coefficient" is intended to clarify. But I think it would be reasonable to have a sub-type of Ratio that only had a single brand, and required that its operands be of that brand.
Making all the functions in `ratio.js` work with both types looks like it would be a chore.